templates/tftmpl/notifier/notifier.go

```diff
@@ -7,6 +7,8 @@ import (
 	"github.com/hashicorp/hcat/dep"
 )
 
+//go:generate mockery --name=Overrider --filename=overrider.go --output=../../../mocks/notifier
```

Comment:
leftovers from development?

Reply:
Nope, this comment allows this generate to be picked up when you run `go generate ./...` from the root of the project. This way that's all you have to do to do all code generations for CTS!

https://go.dev/blog/generate